packages/loader/container-loader/src/test/serializedStateManager.spec.ts

```diff
@@ -176,30 +176,6 @@ describe("serializedStateManager", () => {
 		);
 	});
 
-	it("can't get pending local state when there is no base snapshot", async () => {
```

Comment:
This deletion is intentional?  From a procedural perspective, might be nice to split any material changes into their own PR to go before the tagging.

Reply:
Daniel's got a PR for this change: https://github.com/microsoft/FluidFramework/pull/20054

I added it here to make sure this PR passes with that fix. His change will get merged first.